modules/tools/velero/main.tf

```diff
@@ -0,0 +1,37 @@
+module "velero_namespace" {
+  source = "../../common/namespace"
+
+  resource_limit_cpu    = "100m"
+  resource_limit_memory = "256Mi"
+
+  namespace = var.namespace
+}
+
+resource "helm_release" "velero" {
+  repository = "vmware-tanzu"
+  name       = "velero"
+  chart      = "vmware-tanzu/velero"
+  namespace  = module.velero_namespace.name
+  version    = "2.15.0"
+
+  values = [
+    templatefile("${path.module}/velero-values.tpl", {
+      velero_accesskey_id     = var.velero_aws_access_key_id
+      velero_accesskey_secret = var.velero_aws_secret_access_key
```

Comment:
secrets should be set using `set_sensitive`, otherwise they will appear in plaintext in the console running terraform.  https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release#set_sensitive

Reply:
Have switched to multi-line string using heredoc syntax using set_sensitive, as we discussed.